xapi-rrd.opam

```diff
@@ -9,7 +9,9 @@ tags: [
 ]
 build: [
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name] {with-test}
+]
+build-test: [
+  ["dune" "runtest" "-p" name]
 ]
```

Comment:
The two lines are equivalent and there are discussions about removing `build-test` and just rely on the flags

Reply:
I'm going to merge this because @psafont won't be able to work on this for the next week and we can do this independently.